packages/neuron-wallet/src/controllers/api.ts

```diff
@@ -70,6 +69,32 @@ export default class ApiController {
       e.returnValue = SettingsService.getInstance().locale
     })
 
+    ipcMain.on('get-version', e => {
+      e.returnValue = app.getVersion()
+    })
+
+    ipcMain.on('get-platform', e => {
+      e.returnValue = process.platform
+      // @ts-ignore
+      e.sender.getOwnerBrowserWindow()
```

Comment:
What does it do?

Reply:
I believe that's a copy and paste mistake. 😅